cmd/root.go

```diff
@@ -100,16 +100,71 @@ func WithDialer(d cloudsql.Dialer) Option {
 	}
 }
 
+var longHelp = `
+The Cloud SQL Auth proxy is a utility for ensuring secure connections to your
+Cloud SQL instances. It provides IAM authorization, allowing you to control who
+can connect to your instance through IAM permissions, and TLS 1.3 encryption,
+without having to manage certificates.
+
+For every provided instance connection name, the proxy creates:
+
+- a socket that mimics a database running locally, and
+- an encrypted connection using TLS 1.3 back to your Cloud SQL instance.
+
+The proxy uses an ephemeral certificate to establish a secure connection to your
+Cloud SQL instance. The proxy will refresh those certificates on an hourly
+basis. Existing client connections are unaffected by the refresh cycle.
+
+NOTE: The proxy does not configure the network. You MUST ensure the proxy can
+reach your Cloud SQL instance, either by deploying it in a VPC that has access
+to your Private IP instance, or by configuring Public IP.
+
+To start the proxy, you will need your instance connection name, which may be found
+in the Cloud SQL instance overview page or by using gcloud with the following
+command:
+
+    gcloud sql instances describe INSTANCE --format='value(connectionName)'
+
+Starting the proxy will look like, for example:
```

Comment:
Use the following command to start the proxy. For example, if database connection name is `my-project:us-central:1-my-db-server`: